plugins/source/aws/resources/services/sns/subscriptions.hcl

```
@@ -0,0 +1,55 @@
+//check-for-changes
+service          = "aws"
+output_directory = "."
+add_generate     = true
+
+description_modifier "remove_read_only" {
+  words = ["  This member is required."]
+}
+
+resource "aws" "sns" "subscriptions" {
+  path = "github.com/cloudquery/cloudquery/plugins/source/aws/resources/services/sns.Subscription"
+  ignoreError "IgnoreAccessDenied" {
```

```suggestion
  ignoreError "IgnoreCommonErrors" {
```

Just to make the description match the actual value

---

Updated, thanks